src/stylus/fablevision/breed-button.styl

```diff
@@ -0,0 +1,27 @@
+.geniblocks.breed-button
```

Comment:
For clarity, I would call this `breed-button-surround`, because it's not the actual button.